### PR TITLE
Added JoshuaTestScene

### PR DESCRIPTION
**Pull request by ChillBeanJosh**

### CODE REVIEW

**Review by kyleyang2020 — DISMISSED** [review]
nice added a scene, sorry for the delay

**Review by ChillBeanJosh — COMMENTED** [review]
new

File: Assets/Scripts/PlayerMovement.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    float verticalInput;
+
+    Vector3 moveDirection;
+    Rigidbody rb;
+
+
+    void Start()
+    {
+        rb = GetComponent<Rigidbody>();
+        rb.freezeRotation = true;
```
Comment (TheStress) [inline Assets/Scripts/PlayerMovement.cs:31]: set in editor not in script

File: Assets/Scripts/PlayerMovement.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    private void MyInput()
+    {
+        horizontalInput = Input.GetAxisRaw("Horizontal");
+        verticalInput = Input.GetAxisRaw("Vertical");
+
+    }
+
+    private void MovePlayer()
```
Comment (TheStress) [inline Assets/Scripts/PlayerMovement.cs:62]: `public void Move(Vector3 direction)`

**Pull request closed without merging**